js/main.js

```diff
@@ -1,19 +1,67 @@
-// Функция, возвращающая случайное целое число из переданного диапазона включительно.
+const TITLES = [
+  'Заголовок 1',
+  'Заголовок 2',
+  'Заголовок 3',
+  'Заголовок 4',
+  'Заголовок 5',
+  'Заголовок 6',
+  'Заголовок 7',
+  'Заголовок 8',
+  'Заголовок 9',
+  'Заголовок 10'
+];
 
-const getRandomInt = (min, max) => {
-  min = Math.ceil(min);
-  max = Math.floor(max);
+const TYPE_HOTEL = [
+  'palace',
+  'flat',
+  'house',
+  'bungalow',
+  'hotel'
+];
 
-  if (min >= max || min < 0) {
-    throw new Error('Введите корректное значене: min >= 0 и max > min');
-  }
+const TIMES = [
+  '12:00',
+  '13:00',
+  '14:00',
+];
 
-  return Math.floor(Math.random() * (max - min + 1)) + min;
-};
+const FEATURES = [
+  'wifi',
+  'dishwasher',
+  'parking',
+  'washer',
+  'elevator',
+  'conditioner',
+];
+
+const DESCRIPTION = [
+  'description 1',
+  'description 2',
+  'description 3',
+  'description 4',
+  'description 5',
+  'description 6',
+  'description 7',
+  'description 8',
+  'description 9',
+  'description 10'
+];
+
+const PHOTO = [
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/duonguyen-8LrGtIxxa4w.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/brandon-hoogenboom-SNxQGWxZQi0.jpg',
+  'https://assets.htmlacademy.ru/content/intensive/javascript-1/keksobooking/claire-rendall-b6kAwr1i0Iw.jpg',
+];
+
+const COUNT = 10;
+
+const latMin  = 35.65000;
```

Comment:
Надо исправить: константы мы записываем апперкейсом 
```suggestion
const latMin  = 35.65000;
```
```suggestion
const LAT_MIN  = 35.65000;
```